Dockerfile

```
@@ -495,6 +495,64 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+ARG GOROOT="/home/builder/go"
+ARG GOPATH="${GOROOT}/go"
+
+USER builder
+WORKDIR /home/builder
+COPY ./hashes/govmomi ./hashes
+RUN \
+  sdk-fetch hashes && \
+  tar xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz && \
+  mkdir -p ${GOPATH}/src/github.com/vmware && \
+  mv govmomi-${GOVMOMIVER} ${GOPATH}/src/github.com/vmware/govmomi && \
+  ln -s ${GOPATH}/src/github.com/vmware/govmomi
+
+ARG GOOS="linux"
+ARG CGO_ENABLED=0
+ARG PKGPATH="github.com/vmware/govmomi/govc/flags"
+ARG GOARCH_aarch64="arm64"
+ARG GOARCH_x86_64="amd64"
+ARG GOARCH_ARCH="GOARCH_${ARCH}"
+
+WORKDIR ${GOPATH}/src/github.com/vmware/govmomi
+COPY --chown=0:0 --from=sdk-license-scan /usr/libexec/tools/ /usr/libexec/tools/
+COPY --chown=0:0 --from=sdk-license-scan /home/builder/license-scan/clarify.toml .
+RUN \
+  cp -p LICENSE.txt /usr/share/licenses/govmomi && \
+  go mod tidy && \
+  go mod vendor && \
+  /usr/libexec/tools/bottlerocket-license-scan \
+    --clarify clarify.toml \
+    --spdx-data /usr/libexec/tools/spdx-data \
+    --out-dir /usr/share/licenses/govmomi \
+    go-vendor vendor
+
+WORKDIR ${GOPATH}/src/github.com/vmware/govmomi/govc
+RUN \
+  export GOARCH="${!GOARCH_ARCH}" ; \
+  export PATH="${GOROOT}/bin:${PATH}" ; \
+  export GOFLAGS="-mod=vendor" ; \
+  go build -ldflags " \
+    -X ${PKGPATH}.BuildVersion=${GOVMOMIVER} \
+    -X ${PKGPATH}.BuildCommit=${GOVMOMISHORTCOMMIT} \
+    -X ${PKGPATH}.BuildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
+    "
```

I like the symmetry of the paths elsewhere where the src & dst file paths are the same - this makes the provenance a bit more clear when tracing back through (more explicit!). What do you think about explicitly building the executable into the final path for later `COPY` use:

```suggestion
  mkdir -p /usr/libexec/tools
  go build -mod=vendor -o /usr/libexec/tools/govc -ldflags " \
    -X ${PKGPATH}.BuildVersion=${GOVMOMIVER} \
    -X ${PKGPATH}.BuildCommit=${GOVMOMISHORTCOMMIT} \
    -X ${PKGPATH}.BuildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
    "
```